airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -22,6 +61,108 @@ create table "public"."airbyte_configs_migrations"(
   constraint "airbyte_configs_migrations_pk"
     primary key ("installed_rank")
 );
+create table "public"."connection"(
+  "id" uuid not null,
+  "namespace_definition" namespace_definition_type not null,
+  "namespace_format" varchar(256) null,
+  "prefix" varchar(256) null,
+  "source_id" uuid not null,
+  "destination_id" uuid not null,
+  "name" varchar(256) not null,
+  "catalog" jsonb not null,
+  "status" status_type null,
+  "schedule" jsonb null,
+  "manual" bool not null,
+  "resource_requirements" jsonb null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "connection_pkey"
+    primary key ("id")
+);
+create table "public"."operation"(
+  "id" uuid not null,
+  "workspace_id" uuid not null,
+  "name" varchar(256) not null,
+  "operator_type" operator_type not null,
+  "operator_normalization" jsonb null,
+  "operator_dbt" jsonb null,
+  "tombstone" bool null,
+  "created_at" timestamptz(35) not null,
+  "updated_at" timestamptz(35) not null,
+  constraint "operation_pkey"
+    primary key ("id")
+);
+create table "public"."state"(
+  "id" uuid not null,
+  "connection_id" uuid not null,
+  "state" jsonb null,
```

Comment:
should this field be not null? it is in the yaml file that defines the object currently.

Reply:
In `StandardSyncState` its not marked as required

Reply:
yeah. that makes sense. but if the state is null won't we just not have a record in the state table. like our two options are, when state is null...
1. do not put a record in the state table (then the state column can be not null)
2. put a record in the state table but then have the state column be null

1 is a little clearer for me to think about it. do you have a preference?